features/machine/machine.feature

```diff
@@ -141,4 +141,39 @@ Feature: Machine features testing
     Given I scale the machineset to +2
     Then the machineset should have expected number of running machines
 
+  # @author miyadav@redhat.com
+  # @case_id OCP-24363
+  @admin
+  @destructive
+  Scenario: [MAO] Reconciling machine taints with nodes
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+
+    And I wait for the steps to pass:
+    """
+    Given I clone a machineset named "machineset-24363"
```

Comment:
I think step needs to be renamed `I clone a machineset and name it "([^"]*)"`. Original wording made me thing that machine set `machineset-24363` has been cloned. But it can be done in a future PR for existing scenarios as well.

Reply:
bump

Reply:
I will rename it in a future PR.

Reply:
It is used in multiple test cases , can create a future PR what do you suggest @jhou1